test/task_test.rb

```diff
@@ -613,6 +613,78 @@
         end
     end
 
+    describe "modbus RTU statistics" do
+        it "sends a rtu statistic on startup" do
+            @task.properties.modbus_rtu_statistics_period = Time.at(999)
+            modbus_expect_during_configuration_and_start.to do
+                emit task.start_event
+                have_one_new_sample task.modbus_rtu_statistics_port
+            end
+
+            modbus_expect_execution(@writer, @reader).to do
+                have_no_new_sample(task.modbus_rtu_statistics_port, at_least_during: 2)
+            end
+        end
+
+        it "sends a rtu statistic periodically after 1 sec independently" do
+            @task.properties.modbus_rtu_statistics_period = Time.at(1)
+            modbus_expect_during_configuration_and_start.to do
+                emit task.start_event
+                have_one_new_sample task.modbus_rtu_statistics_port
+            end
+            toc = Time.now
+            expect_execution { have_one_new_sample task.modbus_rtu_statistics_port }
+            expect_execution { have_one_new_sample task.modbus_rtu_statistics_port }
+            tic = Time.now
+
+            assert_in_delta((tic - toc), 2, 0.3)
```

Comment:
```suggestion
            assert_in_delta((tic - toc), 2, 0.9)
```
Maybe bigger than 1 lower then 3 is a better bet here, specially on CI